CHANGELOG.md

```diff
@@ -1,62 +1,92 @@
+## 1.1.1
```

Comment:
Please do not change this file. I will update it whenever we have new release.

Reply:
Sure I'll remove it, do you believe the changelog is good enough?